containers/flatpak/cockpit-client.yml.in

```diff
@@ -16,6 +16,7 @@ modules:
     buildsystem: autotools
     config-opts:
       - --enable-cockpit-client
+      - --enable-pybridge
```

Comment:
I'd like to block this part on https://github.com/cockpit-project/cockpit/labels/pybridge-blocker (same as for e.g. #18757). Fine for the demo of course!

Reply:
It's worth noting that this is more or less a no-op, since we're not using the default rules for building or installing (and don't install the bridge before this at all, anyway).

This basically just gets us the `install-python` rule enabled in the Makefile.